packages/asset-kit-block/src/components/InformationSection.tsx

```diff
@@ -1,6 +1,6 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { hasRichTextValue, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
+import { convertToRTEValue, hasRichTextValue, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
```

Comment:
`convertToRTEValue` should be `convertToRteValue`, see: https://weare.frontify.com/document/42#/frontend-react/feature-file-structure 😬

Reply:
Thanks! Adapted